### PR TITLE
LPD-40171 | LPD-37588 | LPD-37590 | LPD-37594 | LPD-40172 | LPD-40174 | LPD-40176 Allow exporting and importing one Page referencing

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-40171
https://liferay.atlassian.net/browse/LPD-37588
https://liferay.atlassian.net/browse/LPD-37590
https://liferay.atlassian.net/browse/LPD-37594
https://liferay.atlassian.net/browse/LPD-40172
https://liferay.atlassian.net/browse/LPD-40174
https://liferay.atlassian.net/browse/LPD-40176

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:upgrade

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-address-content-web/src/main/java/com/liferay/commerce/address/content/web/internal/portlet/action/EditCommerceAddressMVCRenderCommand.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import com.liferay.portal.kernel.portlet.bridges.mvc.MVCRenderCommand;
 import com.liferay.portal.kernel.security.auth.PrincipalException;
 import com.liferay.portal.kernel.service.CountryService;
+import com.liferay.portal.kernel.service.GroupLocalService;
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-address-content-web/src/main/java/com/liferay/commerce/address/content/web/internal/portlet/action/EditCommerceAddressMVCRenderCommand.java:20]: Make an "update usage" commit with the previous one

File: modules/apps/commerce/commerce-address-content-web/src/main/resources/META-INF/resources/configuration.jsp [hunk 2]
```diff
@@ -28,7 +28,7 @@ CommerceAddressDisplayContext commerceAddressDisplayContext = (CommerceAddressDi
 							<liferay-template:template-selector
 								className="<%= CommerceAddressContentPortlet.class.getName() %>"
 								displayStyle="<%= commerceAddressDisplayContext.getDisplayStyle() %>"
-								displayStyleGroupId="<%= commerceAddressDisplayContext.getDisplayStyleGroupId() %>"
+								displayStyleGroupKey="<%= commerceAddressDisplayContext.getDisplayStyleGroupKey() %>"
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-address-content-web/src/main/resources/META-INF/resources/configuration.jsp:31]: Use the "update usage" commit for this one as well

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@monaco-fabio LGTM overall, just few small changes. Thank you!

**Comment by monaco-fabio** [comment]
ci:test:reports:85156513

**Comment by monaco-fabio** [comment]
ci:report:85156513

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

**Comment by monaco-fabio** [comment]
ci:test:upgrade

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:report:85916778

**Comment by monaco-fabio** [comment]
ci:test:upgrade

**Comment by monaco-fabio** [comment]
ci:report:87333920

**Comment by monaco-fabio** [comment]
ci:test:upgrade

**Comment by monaco-fabio** [comment]
ci:report:87631820

**Comment by alessiorendina** [comment]
@monaco-fabio confirmed that failures are not related to the PR

**Comment by alessiorendina** [comment]
ci:forward:force

**Comment by alessiorendina** [comment]
ci:forward:force

**Comment by alessiorendina** [comment]
Forwarded manually: https://github.com/brianchandotcom/liferay-portal/pull/156831

**Pull request closed without merging**